dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23ExternalContextStartupShutdownTests.java

```diff
@@ -67,6 +68,7 @@ public static void tearDown() throws Exception {
      * @throws Exception
      */
     @Test
+    @SkipForRepeat(SkipForRepeat.EE9_FEATURES)
     public void testExternalContextGetRealPath_Startup_Shutdown() throws Exception {
```

Comment:
If we're skipping this b/c it does not work currently lets get an issue opened for investigation.